Project-Timer-frontend/src/styles.scss

```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+
+html, body { height: 100%; }
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
```

Comment:
sformatuj to  ładnie, dodaj sobie prettiera do projektu albo zainstaluj do VS code

Reply:
done